tests/gui/pages/main.spec.js

```diff
@@ -338,7 +338,7 @@ describe('Browser: MainPage', function() {
           });
 
           settings.set('unmountOnSuccess', false);
-          m.chai.expect(controller.getProgressButtonLabel()).to.equal('50%');
+          m.chai.expect(controller.getProgressButtonLabel()).to.equal('50% Flashing');
```

Comment:
...and there should probably be a test for `makeTitle` too (see my previous comment).